vcd/datasource_vcd_network_isolated.go

```diff
@@ -135,5 +136,16 @@ func datasourceVcdNetworkIsolated() *schema.Resource {
 }
 
 func datasourceVcdNetworkIsolatedRead(d *schema.ResourceData, meta interface{}) error {
+	vcdClient := meta.(*VCDClient)
+
+	_, vdc, err := vcdClient.GetOrgAndVdcFromResource(d)
+	if err != nil {
+		return fmt.Errorf(errorRetrievingOrgAndVdc, err)
+	}
+
+	if vdc.IsNsxt() {
+		fmt.Fprintf(getTerraformStdout(), "WARNING: please use 'vcd_network_isolated_v2' for NSX-T VDCs")
```

Comment:
Shouldn't this be a fatal error instead of a warning?

Reply:
I am a bit afraid that some people might have used `v1` version for NSX-T (which happens to work in some cases even though NSX-T is officially supported only with OpenAPI) and then an upgrade would just break their HCL.